encryption-decryption/src/com/nbugaenco/encryptdecrypt/Main.java

```diff
@@ -0,0 +1,119 @@
+package com.nbugaenco.encryptdecrypt;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class Main {
+
+    private static final Map<Parameter, String> params = new HashMap<>() {{
+        put(Parameter.MODE, "enc");
+        put(Parameter.KEY, "0");
+        put(Parameter.DATA, "");
+        put(Parameter.ALG, "shift");
+        put(Parameter.IN, null);
+        put(Parameter.OUT, null);
+    }};
+
+    public static void main(String[] args) {
+        checkArgs(args);
+
+        EncoderFactory encoderFactory = new EncoderFactory();
+        Encoder encoder = encoderFactory.create(params.get(Parameter.ALG));
+
+        encoder.setKey(Integer.parseInt(params.get(Parameter.KEY)));
+
+        if (params.get(Parameter.DATA) != null && params.get(Parameter.DATA).length() > 0) {
+            encoder.setPrompt(params.get(Parameter.DATA));
+        } else if (params.get(Parameter.IN) != null && params.get(Parameter.IN).length() > 0) {
+            String inPath = params.get(Parameter.IN);
+
+            try {
+                encoder.setPrompt(new String(Files.readAllBytes(Paths.get(inPath))));
```

Comment:
how about ? 

```java
Files.readString(Paths.of(...)) 
```

Reply:
In this case it's returning just array of bytes (byte[]), so I need to make a String from it. Or make a new constructor...